bb-pilcom/bb-pil-backend/templates/relation.hpp.hbs

```diff
@@ -65,6 +57,10 @@ template <typename FF> class {{name}} : public Relation<{{name}}Impl<FF>> {
             }
             return std::to_string(index);
         }
+
+        {{#each labels as |label|}}
+        static constexpr size_t SR_{{label.1}} = {{label.0}};
+        {{/each}}
```

Comment:
i'd throw a comment above this generated block `// Subrelation labels` or something similar. I didn't know what they were at first. I still don't really "know" what a subrelation label is tbh :laughing:

Reply:
Actually I had! hehe. But then even files without labels were getting the comment and I thought that would be confusing. I might add it in a followup PR.